src/Checkbox/Checkbox.less

```diff
@@ -2,22 +2,73 @@
 .moon-checkbox {
 	cursor: pointer;
 
+	.moon-formcheckbox-item &,
+	.moon-toggle-item & {
+		.moon-icon {
+			visibility: hidden;
+			background-color: transparent;
+		}
+		&[checked] .moon-icon {
+			visibility: visible;
+
+			.moon-neutral & {
+				background-color: transparent;
+			}
+		}
+	}
+
 	.moon-icon {
-		visibility: hidden;
 		margin: 0;
 		padding: 0;
+		width: 30px;
+		height: 30px;
+		line-height: 30px;
+		font-size: 60px;
+		border-radius: 15px;
+		color: transparent;
+		background-color: @moon-dark-gray;
+
+		.moon-neutral & {
+			background-color: @moon-white;
+		}
 	}
 	&[checked] .moon-icon {
-		visibility: visible;
+		color: @moon-gray;
+
+		.moon-neutral & {
+			color: @moon-dark-gray;
+		}
 	}
 }
 
 .spotlight {
 	.moon-checkbox[checked]:after {
 		color: @moon-white;
 	}
-	.moon-checkbox:not([checked]):not([disabled]) .moon-icon {
-		visibility: visible;
-		opacity: 0.3;
+
+	&.moon-formcheckbox-item,
+	&.moon-toggle-item {
+		.moon-checkbox:not([disabled]) .moon-icon {
+			background-color: transparent;
+		}
 	}
-}
\ No newline at end of file
+
+	&.moon-toggle-item .moon-checkbox:not([disabled]) {
+		&[checked] .moon-icon {
+			color: @moon-checkbox-toggle-switch-checked-color;
+		}
+		.moon-icon {
+			color: @moon-checkbox-toggle-switch-color;
+		}
+	}
+
+	.moon-checkbox:not([disabled]) {
+		.moon-icon {
+			background-color: rgba(255,255,255,0.41);
```

Comment:
This should be a color variable too. Also, do you think anyone can notice the difference between `0.41` and `0.4` alpha levels? Let's go ahead and round this value down to `0.4`: `rgba(255,255,255,0.4)`.
Alternatively, in the color file, you could use the LESS function to apply the alpha as well: `fade(@moon-white, 40%)`